Dockerfile

```diff
@@ -0,0 +1,14 @@
+# build environment
+FROM node:13.12.0-alpine as builder
+WORKDIR /app
+ENV PATH /app/node_modules/.bin:$PATH
+COPY package.json yarn.lock ./
+RUN yarn install
+COPY . ./
+RUN yarn run build
+
+# production environment
+FROM nginx:stable-alpine
```

Comment:
Better stick to a specific version of Nginx

Reply:
done